### DISCUSSION TITLE
twitter card and opengraph in single HTML output

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#1434] [Q&A] [answered]
Hi,
are there the options to set twitter card and opengraph for single HTML output?
It could be very useful, but it seems to me that we have them only for website output.

Thank you

**Comment by jjallaire** [comment, marked as answer]
There isn't currently a way to do that. My suggestion for this would be to create a "single page website" (i.e. just remove the navbar). We'll definitely improve this in the future but hopefully that gives you a short-term workaround.
